docs/operations-guide/scaling-metabase.md

```diff
@@ -0,0 +1,142 @@
+# Scaling Metabase
+
+Metabase is scalable, battle-tested software used by tens of thousands of companies. It supports high availability via horizontal scaling, and high performance via vertical scaling. Plus it's efficient out of the box: a single core machine with 4 gigs of RAM can scale Metabase to hundreds of users.
+
+This article provides guidance on how to keep Metabase running smoothly in production as the numbers of users and data sources increase. 
+
+We'll cover:
+
+- [**Factors that impact Metabase performance and availability.**](#factors-that-impact-metabase-performance-and-availability)
+- [**Vertical scaling.**](#vertical-scaling) Running a single instance of Metabase with more cores and memory. Vertical scaling can improve Metabase application performance.
+- [**Horizontal scaling.**](#horizontal-scaling) Running multiple instances of Metabase. Horizontal scaling can improve availability/reliability of your Metabase application.
+- [**Data warehouse tuning.**](#data-warehouse-tuning) Way out-of-scope for this article, but we'll discuss some general strategies for data warehousekeeping.
+- [**Metabase application best practices.**](#metabase-application-best-practices) Tactics for improving the performance of questions and dashboards.
+- [**Hosted Metabase.**](#hosted-metabase) Let Metabase handle application operation for you. 
```

Comment:
Requires some more dev effort, but we should also include:
- If this, then that section - if you're experiencing X, scale Metabase by Y
- Hard performance numbers - "assuming a reasonably well configured data warehouse, this is how many simultaneous users this hardware configuration can support".